packages/react-native/src/views/ChatRoomView/ChatRoomView.js

```diff
@@ -4,7 +4,7 @@ import { ChatInput } from '../../components/ChatInput';
 import { MessageActionsSheet } from '../../components/MessageActionsSheet';
 import { MessageList } from '../../components/MessageList';
 import { useRCContext } from '../../contexts/RCInstance';
-import { useMessageStore, useUserStore } from '../../store';
+import { useMessageStore, useChannelStore, useUserStore } from '../../store';
```

Comment:
Plese use `useMemberStore` here, we are storing members and their related info there.

Reply:
done